__tests__/index.test.ts

```diff
@@ -2165,6 +2165,49 @@ describe('Auth0', () => {
     });
   });
 
+  describe('buildLogoutUrl()', () => {
+    it('creates correct query params with empty options', async () => {
+      const { auth0, utils } = await setup();
+
+      auth0.buildLogoutUrl();
+      expect(utils.createQueryParams).toHaveBeenCalledWith({
+        client_id: TEST_CLIENT_ID
+      });
+    });
+
+    it('creates correct query params with `options.client_id` is null', async () => {
+      const { auth0, utils } = await setup();
+
+      auth0.buildLogoutUrl({ client_id: null });
+      expect(utils.createQueryParams).toHaveBeenCalledWith({});
+    });
+
+    it('creates correct query params with `options.client_id` defined', async () => {
+      const { auth0, utils } = await setup();
+
+      auth0.buildLogoutUrl({ client_id: 'another-client-id' });
+      expect(utils.createQueryParams).toHaveBeenCalledWith({
+        client_id: 'another-client-id'
+      });
+    });
+
+    it('creates correct query params with `options.returnTo` defined', async () => {
+      const { auth0, utils } = await setup();
+
+      auth0.buildLogoutUrl({ returnTo: 'https://return.to', client_id: null });
+      expect(utils.createQueryParams).toHaveBeenCalledWith({
+        returnTo: 'https://return.to'
+      });
+    });
+
+    it('creates correct query params when `options.federated` is true', async () => {
+      const { auth0, utils } = await setup();
+
+      auth0.buildLogoutUrl({ federated: true, client_id: null });
+      expect(utils.createQueryParams).toHaveBeenCalledWith({});
+    });
+  });
```

Comment:
These tests all exist as part of the `logout` tests, I think we should remove those and keep the ones you added.